matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/signout/SignOutService.kt

```diff
@@ -24,7 +24,7 @@ import org.matrix.android.sdk.api.auth.data.Credentials
 interface SignOutService {
 
     /**
```

Comment:
<!--
  0 failure: 
  1 warning:  The first sentenc...
  
  
  DangerID: danger-id-Danger;
  File: matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/signout/SignOutService.kt;
  Line: 26;
-->

- :warning: The first sentence of this KDoc does not end with the correct punctuation.